sdk/typescript/test/e2e/utils/setup.ts

```diff
@@ -116,10 +116,9 @@ export async function publishPackage(
   const cap = tx.publish(
     compiledModules.map((m: any) => Array.from(fromB64(m))),
   );
-  tx.moveCall({
-    target: '0x2::package::make_immutable',
-    arguments: [cap],
-  });
+
+  // Make the package upgradeable later on
```

Comment:
Not sure I understand this comment -- the act of not burning the `UpgradeCap` makes this package upgradeable.

Reply:
What I was meaning to say in this comment is "we're transferring the `cap` to the sender so this package can be upgraded later on" (i.e., this package is upgradeable). I'll rephrase/make this clearer.